qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -105,15 +105,21 @@
             Windows:
                 gagent_guest_reboot_pattern = ""
             s390x:
-                gagent_guest_reboot_pattern = "System Reboot"
+                RHEL.9:
+                    gagent_guest_reboot_pattern = "System Reboot"
```

Comment:
Can we leave the RHEL.9 pattern as the default one for s390x? So if there is a RHEL.10 guest in the future, and if the pattern works, we don't need to add it anymore. The new code likes following:

            s390x:
                    gagent_guest_reboot_pattern = "System Reboot"
                    RHEL.8:
                              gagent_guest_reboot_pattern = "Started Reboot"

This also applies to the shutdown pattern in line 120.

Reply:
Sure, here ais the updates, could you please check?

Reply:
LGTM.